java/ql/src/experimental/Security/CWE/CWE-555/CredentialsInPropertiesFile.qhelp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>
+      Credentials management issues occur when credentials are stored in plaintext in 
+      an application’s properties file. Common credentials include but are not limited 
```

Comment:
```suggestion
      an application's properties file. Common credentials include but are not limited 
```
This is currently a single quote, it should be an apostrophe.

Reply:
Thanks @Marcono1234 for reviewing this PR. I've made the change.